src/std/misc/string.ss

```diff
@@ -0,0 +1,64 @@
+;; -*- Gerbil -*-
+package: utils
+;;;; String utilities
+
+(export
+  string-split-suffix
+  string-trim-suffix
+  string-split-eol
+  string-trim-eol
+  +cr+ +lf+ +crlf+)
+
+(import
+  :std/srfi/13)
+
+
+;; TODO: find a welcoming home for this macro.
+(defrules first-value ()
+  ((_ form) (with ((values x . _) form) x)))
+
+;; Split a string based on the given suffix, if present.
+;; Return three values:
+;; - the trimmed string,
+;; - the suffix (eq? to the argument) if found, or an empty string if not found, and
+;; - a boolean indicating whether the suffix was found or not.
```

Comment:
Here too, is there any utility in the 3rd value?

Reply:
Not to have to test for equality and/or to distinguish between success and failure when the suffix is "".

Reply:
I would rather have a single value version that just trims; we can have a multi-valued version as well, but most of the time you don't care about the chomped part (or whether any chomping occured)